### PR TITLE
Format code with prettier

### PR DESCRIPTION
**Pull request by SimenB**
I love prettier. 😄 Makes it so easy to have consistent styling, and it allows us to remove some eslint rules as well

### CODE REVIEW

File: lib/pushgateway.js [hunk 1]
```diff
@@ -63,19 +70,26 @@ function useGateway(method, job, groupings, callback) {
...
 function generateGroupings(groupings) {
-	if(!groupings) {
+	if (!groupings) {
 		return '';
 	}
-	return Object.keys(groupings).map(function(key) {
-		return ['/', encodeURIComponent(key), '/', encodeURIComponent(groupings[key])].join('');
-	}).join('');
+	return Object.keys(groupings)
+		.map(function(key) {
```
Comment (siimon) [inline lib/pushgateway.js]: Not so much about Prettier, but we could use templated strings instead of Array.join

File: lib/registry.js [hunk 2]
```diff
@@ -48,14 +47,19 @@ var registry = (function() {
...
 			.map(this.getMetricAsPrometheusString, this)
 			.join('\n');
 	};
 
 	Registry.prototype.registerMetric = function(metricFn) {
-		if(_metrics[metricFn.name] && _metrics[metricFn.name] !== metricFn) {
-			throw new Error('A metric with the name ' + metricFn.name + ' has already been registered.');
+		if (_metrics[metricFn.name] && _metrics[metricFn.name] !== metricFn) {
+			throw new Error(
+				'A metric with the name ' +
```
Comment (siimon) [inline lib/registry.js]: Same here, templated strings is better

File: lib/validation.js [hunk 3]
```diff
@@ -22,8 +22,13 @@ exports.validateLabelName = function(names) {
 
 exports.validateLabel = function validateLabel(savedLabels, labels) {
 	Object.keys(labels).forEach(function(label) {
-		if(savedLabels.indexOf(label) === -1) {
-			throw new Error('Added label "' + label + '" is not included in initial labelset: ' + util.inspect(savedLabels));
+		if (savedLabels.indexOf(label) === -1) {
+			throw new Error(
```
Comment (siimon) [inline lib/validation.js]: Templated string 🙂

**Comment by SimenB** [comment]
I'll send a separate PR for template strings 🙂

**Pull request merged**